### ISSUE TITLE
make should not attempt to compile vorbis plugin if devel package is not installed

### ISSUE THREAD

**Issue by Oleksiy-Yakovenko** [DeaDBeeF-Player/deadbeef#820]
Original [issue 918](https://code.google.com/p/ddb/issues/detail?id=918) created by Alexey-Yakovenko on 2012-12-06T18:28:28.000Z:

Hi, this is an issue if you compile from source.
(Please note I don't need &quot;help&quot; what to do, it's just that it should not happen.)

$ ./configure --enable-vorbis

Supposed you FORGOT to install libvorbis-dev.

Then this should NOT happen:

make  all-recursive
make[1]: Entering directory `/media/sdh7/d2/deadbeef-0.5.6'
Making all in pixmaps
make[2]: Entering directory`/media/sdh7/d2/deadbeef-0.5.6/pixmaps'
make[2]: Nothing to be done for `all'.
make[2]: Leaving directory`/media/sdh7/d2/deadbeef-0.5.6/pixmaps'
Making all in icons
make[2]: Entering directory `/media/sdh7/d2/deadbeef-0.5.6/icons'
make[2]: Nothing to be done for`all'.
make[2]: Leaving directory `/media/sdh7/d2/deadbeef-0.5.6/icons'
Making all in plugins/lastfm
make[2]: Entering directory`/media/sdh7/d2/deadbeef-0.5.6/plugins/lastfm'
  CC       lastfm.lo
  CCLD     lastfm.la
make[2]: Leaving directory `/media/sdh7/d2/deadbeef-0.5.6/plugins/lastfm'
Making all in plugins/mpgmad
make[2]: Entering directory`/media/sdh7/d2/deadbeef-0.5.6/plugins/mpgmad'
  CC       mpgmad.lo
  CCLD     mpgmad.la
make[2]: Leaving directory `/media/sdh7/d2/deadbeef-0.5.6/plugins/mpgmad'
Making all in plugins/vorbis
make[2]: Entering directory`/media/sdh7/d2/deadbeef-0.5.6/plugins/vorbis'
  CC       vorbis.lo
  CC       vcedit.lo
  CCLD     vorbis.la
gcc: error: /usr/lib/libvorbisfile.so: No such file or directory
gcc: error: /usr/lib/libvorbis.so: No such file or directory
make[2]: **\* [vorbis.la] Error 1
make[2]: Leaving directory `/media/sdh7/d2/deadbeef-0.5.6/plugins/vorbis'
make[1]: *** [all-recursive] Error 1
make[1]: Leaving directory`/media/sdh7/d2/deadbeef-0.5.6'
make: **\* [all] Error 2

2 alternatives:
- Auto-disable vorbis and WARN if configure finds out that you don't have it installed (or only the runtine, not the dev package)
- Interrupt configure if --enable-vorbis was specified and no libvorbis-dev found.

In other words, in such case it must not get to the point as above but bail out way earlier.

With other plugins, it works fine
(i. e. if you miss yasm you will get 'ffap' auto-disabled even though you explicitly specified --enable-ffap. This is correct behavior.)
